Lib/test/test_pdb.py

```diff
@@ -1443,6 +1443,19 @@ def test_issue16180(self):
             'Fail to handle a syntax error in the debuggee.'
             .format(expected, stdout))
 
+    def test_issue26053(self):
+        # run command of pdb prompt echoes the correct args
+        script = "print('hello')"
+        commands = """
+            continue
+            run a b c
+            run d e f
+            quit
+        """
+        stdout, stderr = self.run_pdb_script(script, commands)
+        output = ';'.join([x.strip() for x in stdout.splitlines()])
+        self.assertIn("Restarting main.py with arguments:;a b c", output)
+        self.assertIn("Restarting main.py with arguments:;d e f", output)
```

Comment:
I find the use of ';' here rather confusing, why not '\n'?

Reply:
You're right, \n would be consistent with test_errors_in_command